tests/Fastapi/test_fastapi.py

```diff
@@ -524,3 +539,87 @@ def test_fastapi_root_path(fastapi_root_path: str):
     # The API should migrate (and return 404 here)
     response = test_client.get("/auth/signup/email/exists?email=test@example.com")
     assert response.status_code == 404
+
+
+@mark.asyncio
+@mark.parametrize("token_transfer_method", ["cookie", "header"])
+async def test_should_clear_all_response_during_refresh_if_unauthorized(
```

Comment:
We should add this test for other frameworks if possible.

Reply:
Other tests seem to be replicated for each framework, so why not these?

Reply:
Good point. Other frameworks (Django/Flask) seems to have fewer and dedicated tests unlike FastAPI (it has tons of them). 

So based on your point, I'm adding test for what we have changed in django/flask framework: BaseResponse `delete_header` method. See `test_remove_header_works` in `test_django.py` and `test_flask.py`